src/styles.css

```diff
@@ -1,4 +1,38 @@
-html, body, #main-view {
+html, body {
   height: 100%;
   width: 100%;
 }
+
+body {
+  background: #FAFAFA;
+  font-family: "Source Sans Pro";
+}
+
+
+#main-view {
```

Comment:
I usually like to tie styles as closely to the thing they are styling as I can, so this would be in a file called `MainView.css`, but for an app this small, it might not matter.